Makefile

```diff
@@ -1,22 +1,12 @@
 TAG = $(shell gitmeta image tag)
 REPO = autonomy/cluster-api-provider-talos
-# Image URL to use all building/pushing image targets
-IMG ?= $(REPO):$(TAG)
 
-all: test manager
+all: test docker-build
 
 # Run tests
 test:
 	docker build . --target $@ -t $(REPO):test
 
-# Build manager binary
-manager: generate fmt vet
```

Comment:
Unneeded. This is essentially our build step in the dockerfile.

Reply:
We need this to build locally no?